Assets/Scripts/DoorController.cs

```diff
@@ -47,4 +54,32 @@ public override uint GetUseTime(PlayerInteractionController Caller)
     {
         return interactionTime;
     }
+    private bool IsPlayerInBounds(GameObject Player)
+    {
+        BoxCollider2D playerCollider = Player.GetComponent<BoxCollider2D>();
+        if (standCollider.bounds.Intersects(playerCollider.bounds))
+        {
+            Debug.Log("You're standing in the fucking door you wanker");
+            return true;
+        }
+        else
+        {
+            Debug.Log("Good Job you're out of the bounds of the door youre still a cunt");
+            return false;
+        }
+
+    }
+    public override List<Sprite> GetSprite(PlayerInteractionController Caller)
+    {
+        List <Sprite> sprites = new List<Sprite>();
+        if (IsPlayerInBounds(Caller.gameObject))
+        {
+            sprites.Add(ForbiddenSprite);
```

Comment:
For consideration: we have some logic that the interactable item may be disabled. What if we move this "can use/can't use icon" logic there?